docs/governance/tokenomics.md

```diff
@@ -2,200 +2,171 @@
+A **validator** can also add **additional stake** to it via the `cudos-noded CLI` to increase its voter power.
 
-If a validator signs more than one block at the same height, that's also an offence punishable by slashing 5% of their tokens.
+## Block Rewards 
 
-This could be the case if someone duplicates their validator and try to double their voting power.
+The more tokens staked to a **Validator node**, the higher the likelihood of it being chosen to propose and sign a block. 
 
-Slashed tokens are sent to the community pool.
+* **Block rewards** are calculated after a block is issued. 
 
+:::info Block rewards
 
-## Transactions 
+*Block rewards* =  *staking rewards* + *tx fees* 
+:::
 
-How to view the mempool? Or pending transactions 
-If you're a validator you can keep your mempool to yourself when it is your turn to propose a block then you can submit your own transactions. 
+* Staking rewards occur in a **single pool** and are distributed as follows:  
 
-If you sync your own node with other nodes then you can view pending transactions.
+    1. Validators are assigned rewards proportional to their 
+    
+    2. Users who have delegated tokens are required to **actively** claim those rewards by sending appropriate txs to the blockchain.
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+    3. The **block proposer** gets a base bonus as 1% of the block rewards
+    * In addition, it gets (4*P)% percent of the block rewards where P=(total power of validators with included precommits / total bonded validator power)
+    * This means it gets an additional bonus between 2.67% and 4%
+    * This means the block proposer’s bonus is between 3.67% and 5% of the total block rewards.
 
-* We have a min-gas-price parameter set at 0.000005 CUDOS below which txs won’t be accepted into mempools
+    4. 20% goes to the community pool.
 
-    This parameter is local to each node on the network, can be changed off-chain, and cannot be queried on-chain.
+    5. The rest is distributed to all of the bonded validators (including the block proposer) proportional to their voting power, regardless of them signing the block.
 
-    Hence we must rely on the community for this implementation to work properly.
 
-* Full-nodes that keep these mempools must have a mechanism to protect themselves from over-usage.
+## Distribution Schedule
 
-:::warning 
-Default `gas-wanted on the CLI is 200,000`. Hence the fee is calculated based on this. Users must be aware of this. Be careful to specify this as what you are prepared to pay. 
+**Tx fees** should be the main source of revenue for **validators** and **delegators**. However, due to the expectation of low early engagement numbers on the **Cudos network**, additional rewards are provided to users. 
```

Comment:
I think this is more relevant as an explanation of what staking rewards are. And we've already been talking about staking rewards, so I don't think this is a good place for this. I'd say we should either move this to the top where staking rewards is mentioned for the first time, or simply remove it.

Reply:
Makes sense. Done